ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -192,6 +209,41 @@ pub struct CInitOptions {
     pub density: f32,
     pub vr_pointer: *mut c_void,
     pub enable_subpixel_text_antialiasing: bool,
+    pub vslogger_mod_list: *mut *mut c_char,
```

Comment:
This can be `*const *const` instead, since we don't mutate the values that are pointed at.

Reply:
Correct.  I originally had it as const * const*.  Forgot to change it back after I did some testing.
Implemented.

Reply:
Sorry, meant *const *const....

Reply:
Implemented.